src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -179,6 +179,9 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                         services.AddSingleton<RazorCodeActionResolver, ExtractToCodeBehindCodeActionResolver>();
                         services.AddSingleton<RazorCodeActionResolver, CreateComponentCodeActionResolver>();
 
+                        // Refactoring
```

Comment:
I'd just put this under `Other` until we have some more `Refactoring` services. Don't feel too strongly though.